ui/containers/commands.go

```diff
@@ -68,7 +68,7 @@ func (cnt *Containers) exec() {
 
 	sessionID, err := cnt.execTerminalDialog.PrepareForExec(cnt.selectedID, cnt.selectedName, &execOpts)
 	if err != nil {
-		msg := fmt.Sprintf("CONTAINER EXEC ERROR (%s)", cnt.selectedID)
+		msg := fmt.Sprintf("CONTAINER EXEC ERROR (%s): %s", cnt.selectedID, err.Error())
```

Comment:
A better way to do this is
fmt.Sprintf("CONTAINER EXEC ERROR (%s): %v", cnt.selectedID, err)

Reply:
Sure, will update error message dialog with following format for the rest as all.

Reply:
Update all and create pull request [#6 ](https://github.com/containers/podman-tui/pull/6)